controllers/task.js

```diff
@@ -70,6 +70,24 @@ const insertUserTask = async (req, res, next) => {
 		}
 	}
 
+	let payload = {
+		name:process.env.DEFAULT_SECTION,
+		projectId:data.projectId
+	}
+	let sectionfind = await ProjectSections.findSection(payload)
+
+	if(sectionfind.id == data.section){
```

Comment:
if(sectionfind.id == data.section){
		data.ratingAllowed = false
		if(!data.miscType){
			return res.status(400).send(sendResponse(401, 'Type is required', 'insertUserTask', null, req.data.signature))
		}

	}else{
		data.ratingAllowed = true
		if(data.miscType){
			return res.status(400).send(sendResponse(401, 'Type is Not required', 'insertUserTask', null, req.data.signature))
		}
	}

no need to write whole if else instead write like this



		data.ratingAllowed = false
	if(!(sectionfind.id == data.section)){
		data.ratingAllowed = true
}
		if(!data.miscType){
			return res.status(400).send(sendResponse(401, 'Type is required', 'insertUserTask', null, req.data.signature))
		}